pkg/sql/logictest/tests/5node/5node_test: TestLogic_distsql_stats failed

pkg/sql/logictest/tests/5node/5node_test.TestLogic_distsql_stats [failed](https://mesolite.cluster.engflow.com/invocations/default/a94f8436-3423-4968-b599-2bf3b558e8a1?testReportRun=1&testReportShard=12&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy81bm9kZTo1bm9kZV90ZXN0) on master @ [1283b87ce2161807660263733659e3de15ff5faa](https://github.com/cockroachdb/cockroach/commits/1283b87ce2161807660263733659e3de15ff5faa):


```
[23:24:28] --- progress: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 374 statements
    logic.go:3205: let $hist_crdb_internal_idx_expr = 1097915002105331717
    logic.go:3205: let $hist_d = 1097915002551664645
    logic.go:3205: let $hist_e = 1097915002716454917
    logic.go:3205: let $hist_crdb_internal_idx_expr = 1097915002871578629
    logic.go:3205: let $hist_id_timestamp_outer_buckets_full = 1097915003967471621
[23:25:06] --- progress: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 431 statements
    logic.go:3205: let $job = 1097915005514809345
[23:25:27] --- progress: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 436 statements
    logic.go:3143: 
         
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3414: EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 9
        expected:
            distribution: full
            vectorized: true
            ·
            • render
            │
            └── • filter
                │ estimated row count: 1
                │ filter: n = 1
                │
                └── • scan
                      estimated row count: 18 (1.8% of the table; stats collected <hidden> ago)
                      table: mno@mno_o_idx
                      spans: [/9 - /9]
            
        but found (query options: "retry") :
            distribution: local
            vectorized: true
            ·
            • render
            │
            └── • filter
                │ estimated row count: 1
                │ filter: n = 1
                │
                └── • scan
                      estimated row count: 18 (1.8% of the table; stats collected <hidden> ago)
                      table: mno@mno_o_idx
                      spans: [/9 - /9]
            
[23:26:12] --- progress: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 441 statements
[23:26:12] --- done: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats with config 5node: 441 tests, 1 failures
    logic.go:4372: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3431: error while processing
    logic.go:4372: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3431: too many errors encountered, skipping the rest of the input
    panic.go:636: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_distsql_stats3477672857
--- FAIL: TestLogic_distsql_stats (137.58s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=12</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_distsql_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-53488

pkg/sql/logictest/tests/5node/5node_test.TestLogic_distsql_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20342643?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20342643?buildTab=artifacts#/) on master @ [54bcb718d8801123e297b47c0b00dfb016fdc1a0](https://github.com/cockroachdb/cockroach/commits/54bcb718d8801123e297b47c0b00dfb016fdc1a0):


```
    logic.go:3205: let $hist_crdb_internal_idx_expr = 1098040447310692356
    logic.go:3205: let $hist_d = 1098040448442466308
    logic.go:3205: let $hist_e = 1098040448845053956
    logic.go:3205: let $hist_crdb_internal_idx_expr = 1098040449329102852
    logic.go:3205: let $hist_id_timestamp_outer_buckets_full = 1098040451809017860
[10:02:33] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16778/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 414 statements
[10:02:45] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16778/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 431 statements
    logic.go:3205: let $job = 1098040455349043201
[10:02:48] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16778/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 436 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16778/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3414: EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 9
        expected:
            distribution: full
            vectorized: true
            ·
            • render
            │
            └── • filter
                │ estimated row count: 1
                │ filter: n = 1
                │
                └── • scan
                      estimated row count: 18 (1.8% of the table; stats collected <hidden> ago)
                      table: mno@mno_o_idx
                      spans: [/9 - /9]
            
        but found (query options: "retry") :
            distribution: local
            vectorized: true
            ·
            • render
            │
            └── • filter
                │ estimated row count: 1
                │ filter: n = 1
                │
                └── • scan
                      estimated row count: 18 (1.8% of the table; stats collected <hidden> ago)
                      table: mno@mno_o_idx
                      spans: [/9 - /9]
            
[10:03:34] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16778/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 441 statements
[10:03:34] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16778/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats with config 5node: 441 tests, 1 failures
    logic.go:4372: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16778/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3431: error while processing
    logic.go:4372: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16778/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3431: too many errors encountered, skipping the rest of the input
    panic.go:636: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/7ac5d9b3004e849e24aae92719f60585/logTestLogic_distsql_stats2751871212
--- FAIL: TestLogic_distsql_stats (111.11s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_distsql_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/5node/5node_test.TestLogic_distsql_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20342709?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20342709?buildTab=artifacts#/) on master @ [54bcb718d8801123e297b47c0b00dfb016fdc1a0](https://github.com/cockroachdb/cockroach/commits/54bcb718d8801123e297b47c0b00dfb016fdc1a0):


```
    logic.go:3205: let $hist_crdb_internal_idx_expr = 1098051480021630981
    logic.go:3205: let $hist_d = 1098051480482086917
    logic.go:3205: let $hist_e = 1098051480671944709
    logic.go:3205: let $hist_crdb_internal_idx_expr = 1098051480858591237
    logic.go:3205: let $hist_id_timestamp_outer_buckets_full = 1098051481874202629
[10:58:38] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 412 statements
[10:58:53] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 431 statements
    logic.go:3205: let $job = 1098051483558805505
[10:59:06] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 436 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3414: EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 9
        expected:
            distribution: full
            vectorized: true
            ·
            • render
            │
            └── • filter
                │ estimated row count: 1
                │ filter: n = 1
                │
                └── • scan
                      estimated row count: 18 (1.8% of the table; stats collected <hidden> ago)
                      table: mno@mno_o_idx
                      spans: [/9 - /9]
            
        but found (query options: "retry") :
            distribution: local
            vectorized: true
            ·
            • render
            │
            └── • filter
                │ estimated row count: 1
                │ filter: n = 1
                │
                └── • scan
                      estimated row count: 18 (1.8% of the table; stats collected <hidden> ago)
                      table: mno@mno_o_idx
                      spans: [/9 - /9]
            
[10:59:52] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 441 statements
[10:59:52] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats with config 5node: 441 tests, 1 failures
    logic.go:4372: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3431: error while processing
    logic.go:4372: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3431: too many errors encountered, skipping the rest of the input
    panic.go:636: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/7ac5d9b3004e849e24aae92719f60585/logTestLogic_distsql_stats952929105
--- FAIL: TestLogic_distsql_stats (102.38s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_distsql_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/5node/5node_test.TestLogic_distsql_stats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20345498?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20345498?buildTab=artifacts#/) on master @ [f5c0bbdfd3cde3556340a67d7d910945ed961e99](https://github.com/cockroachdb/cockroach/commits/f5c0bbdfd3cde3556340a67d7d910945ed961e99):


```
    logic.go:3205: let $hist_crdb_internal_idx_expr = 1098154559096586243
    logic.go:3205: let $hist_d = 1098154559711182851
    logic.go:3205: let $hist_e = 1098154559954714627
    logic.go:3205: let $hist_crdb_internal_idx_expr = 1098154560173277187
[19:42:56] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/165/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 399 statements
    logic.go:3205: let $hist_id_timestamp_outer_buckets_full = 1098154561461846019
[19:43:31] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/165/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 431 statements
    logic.go:3205: let $job = 1098154563562962945
[19:43:52] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/165/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 436 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/165/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3414: EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 9
        expected:
            distribution: full
            vectorized: true
            ·
            • render
            │
            └── • filter
                │ estimated row count: 1
                │ filter: n = 1
                │
                └── • scan
                      estimated row count: 18 (1.8% of the table; stats collected <hidden> ago)
                      table: mno@mno_o_idx
                      spans: [/9 - /9]
            
        but found (query options: "retry") :
            distribution: local
            vectorized: true
            ·
            • render
            │
            └── • filter
                │ estimated row count: 1
                │ filter: n = 1
                │
                └── • scan
                      estimated row count: 18 (1.8% of the table; stats collected <hidden> ago)
                      table: mno@mno_o_idx
                      spans: [/9 - /9]
            
[19:44:37] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/165/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats: 441 statements
[19:44:37] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/165/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats with config 5node: 441 tests, 1 failures
    logic.go:4372: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/165/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3431: error while processing
    logic.go:4372: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/165/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/distsql_stats:3431: too many errors encountered, skipping the rest of the input
    panic.go:636: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/7ac5d9b3004e849e24aae92719f60585/logTestLogic_distsql_stats535225748
--- FAIL: TestLogic_distsql_stats (132.08s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_distsql_stats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/logictest/testdata/logic_test/distsql_stats
+++ b/pkg/sql/logictest/testdata/logic_test/distsql_stats
@@ -1,6 +1,6 @@
 # LogicTest: 5node
 
-skip #151839
+skip under race
 
 # Disable histogram collection.
 statement ok
@@ -3411,76 +3411,36 @@ NULL  {m}  1000  1000  true
 NULL  {n}  1000  50    true
 NULL  {o}  1000  33    true
 
+# All we care about is the row counts, so don't error if other parts of the explain output change
 query T retry
-EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 9
+SELECT info FROM [EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 9] WHERE info LIKE '%estimated row count:%'
 ----
-distribution: full
-vectorized: true
-·
-• render
-│
-└── • filter
     │ estimated row count: 1
-    │ filter: n = 1
-    │
-    └── • scan
           estimated row count: 18 (1.8% of the table; stats collected <hidden> ago)
-          table: mno@mno_o_idx
-          spans: [/9 - /9]
 
+# All we care about is the row counts, so don't error if other parts of the explain output change
 query T
-EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 11
+SELECT info FROM [EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 11] WHERE info LIKE '%estimated row count:%'
 ----
-distribution: full
-vectorized: true
-·
-• render
-│
-└── • filter
     │ estimated row count: 1
-    │ filter: sqrt(m::FLOAT8)::INT8 = 11
-    │
-    └── • scan
           estimated row count: 20 (2.0% of the table; stats collected <hidden> ago)
-          table: mno@mno_n_idx
-          spans: [/1 - /1]
 
 statement ok
 SET optimizer_use_virtual_computed_column_stats = false
 
+# All we care about is the row counts, so don't error if other parts of the explain output change
 query T
-EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 9
+SELECT info FROM [EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 9] WHERE info LIKE '%estimated row count:%'
 ----
-distribution: full
-vectorized: true
-·
-• render
-│
-└── • filter
     │ estimated row count: 7
-    │ filter: n = 1
-    │
-    └── • scan
           estimated row count: 10 (1.0% of the table; stats collected <hidden> ago)
-          table: mno@mno_o_idx
-          spans: [/9 - /9]
 
+# All we care about is the row counts, so don't error if other parts of the explain output change
 query T
-EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 11
+SELECT info FROM [EXPLAIN SELECT * FROM mno WHERE n = 1 AND o = 11] WHERE info LIKE '%estimated row count:%'
 ----
-distribution: full
-vectorized: true
-·
-• render
-│
-└── • filter
     │ estimated row count: 7
-    │ filter: n = 1
-    │
-    └── • scan
           estimated row count: 10 (1.0% of the table; stats collected <hidden> ago)
-          table: mno@mno_o_idx
-          spans: [/11 - /11]
 
 # Regression for not setting the TypeResolver on the SemaContext when dealing
 # with stats on virtual computed columns (#122312).
@@ -3577,19 +3537,12 @@ column_names  row_count  null_count  distinct_count  avg_size  has_histogram
 {b}           2          0           2               8         true
 {c}           2          0           2               8         true
 
+# All we care about is the row counts, so don't error if other parts of the explain output change
 query T retry
-EXPLAIN SELECT count(*) FROM t138809 WHERE b > 1
+SELECT info FROM [EXPLAIN SELECT count(*) FROM t138809 WHERE b > 1] WHERE info LIKE '%estimated row count:%'
 ----
-distribution: full
-vectorized: true
-·
-• group (scalar)
 │ estimated row count: 1
-│
-└── • scan
       estimated row count: 2 (100% of the table; stats collected <hidden> ago)
-      table: t138809@t138809_pkey
-      spans: [/1 - ]
 
 statement ok
 SET CLUSTER SETTING jobs.debug.pausepoints = 'schemachanger.root..1'
@@ -3597,19 +3550,12 @@ SET CLUSTER SETTING jobs.debug.pausepoints = 'schemachanger.root..1'
 statement error job \d+ was paused before it completed with reason: pause point "schemachanger\.root\.\.1" hit
 ALTER TABLE t138809 DROP COLUMN c
 
+# All we care about is the row counts, so don't error if other parts of the explain output change
 query T
-EXPLAIN SELECT count(*) FROM t138809 WHERE b > 1
+SELECT info FROM [EXPLAIN SELECT count(*) FROM t138809 WHERE b > 1] WHERE info LIKE '%estimated row count:%'
 ----
-distribution: full
-vectorized: true
-·
-• group (scalar)
 │ estimated row count: 1
-│
-└── • scan
       estimated row count: 2 (100% of the table; stats collected <hidden> ago)
-      table: t138809@t138809_pkey
-      spans: [/1 - ]
 
 statement ok
 SET CLUSTER SETTING jobs.debug.pausepoints = DEFAULT

